src/agent/rustjail/src/cgroups/systemd/dbus_client.rs

```diff
@@ -36,8 +36,9 @@ pub struct DBusClient {}
 
 impl DBusClient {
     fn build_proxy(&self) -> Result<SystemManager<'static>> {
-        let connection = zbus::blocking::Connection::system()?;
-        let proxy = SystemManager::new(&connection)?;
+        let connection =
+            zbus::blocking::Connection::system().context("Establishing a D-Bus connexion")?;
```

Comment:
Nit : `connexion` is French wording for `connection`

Reply:
Ach, right. Will fix the spelling.